ptflops/flops_counter.py

```diff
@@ -366,6 +366,63 @@ def batch_counter_hook(module, input, output):
     module.__batch_counter__ += batch_size
 
 
+def rnn_flops(flops, rnn_module, w_ih, w_hh, input_size):
+    # matrix matrix mult ih state and internal state
+    flops += w_ih.shape[0]*(2*w_ih.shape[1] - 1)
```

Comment:
For consistency with the other hooks it's better to keep just `w_ih.shape[0]*w_ih.shape[1]` here.
Current variant counts flops (* and + as separate operations), while the other hooks count amount of FMA operations.

Reply:
Ok, so it is a MAC computation. But how are summations accounted for? I mean, for example, adding a bias or adding two tensors. Because I have added this as an operation. Should this be deleted or maintained?

Reply:
Let's keep this stuff as is. I prefer to aggregate * and + only within single imaginary lapack routine, even if in the perfect case it's possible to use FMA (like when we need to add a vector to a result of hadamard product).

Reply:
Ok. For me it is no problem. I change it and update it.